caravel/assets/javascripts/explorev2/reducers.js

```diff
@@ -0,0 +1,127 @@
+import shortid from 'shortid';
+import * as actions from './actions';
+
+const defaultTimeFilter = {
+  timeColumn: null,
+  timeGrain: null,
+  since: null,
+  until: null,
+};
+
+const defaultGroupBy = {
+  groupByColumn: [],
+  metrics: [],
+};
+
+const defaultSql = {
+  where: '',
+  having: '',
+};
+
+const defaultFilter = {
+  id: shortid.generate(),
+  eq: null,
+  op: 'in',
+  col: null,
+};
+
+export const initialState = {
```

Comment:
i mean put initial state in a file called `store.js`

then rather than importing initial state from `./reducers.js`, import from `store.js`

the way you are creating the store looks good.